README.md

```diff
@@ -141,23 +121,90 @@ On both your PC and your robot, clone and install the package:
 
 ```bash
 git clone git@github.com:hello-robot/stretch_ai.git --recursive
+```
+
+#### Install On PC
 
-# Install
+The installation script will install the package and its dependencies, as well as (optionally) some perception modules.
+
+```bash
 cd stretch_ai
+./install.sh
+```
+
+#### Install On the Robot
+
+Robot installation can be tricky, because we use some features from [ROS2](https://docs.ros.org/en/humble/index.html), specifically the [Nav2](https://github.com/ros-navigation/navigation2) package for LIDAR slam.
+
+You will need to link Stretch AI into your ROS workspace. There are two ways to do this; either install stretch AI in your base python environment, or link the conda environment into ROS (advanced). Either way, you will then need to [set up the ROS2 bridge](#set-up-ament-workspace) in your Ament workspace.
+
+*Why all this complexity?* We run a set of ROS2 nodes based on the [HomeRobot](https://github.com/facebookresearch/home-robot) and [OK-Robot](https://ok-robot.github.io/) codebases for mobile manipulation and localization. In particular, this allows us to use [Nav2](https://docs.nav2.org/), a very well-tested ROS2 navigation stack, for localization, which makes it easier to build complex applications. You do not need to understand ROS2 to use this stack.
+
+##### Option 1: Install Stretch AI in Base Python Environment
+
+To install in the base python environment, you need to make sure build tools are up to date:
+
+```bash
+conda deactivate  # only if you are in a conda environment
+pip install --upgrade pip setuptools packaging build meson ninja
+```
+
+This is particularly an issue for scikit-fmm, which is used for motion planning. After this is done, you can install the package as normal:
+
+```bash
 pip install ./src
 ```
 
+Then, [set up the ROS2 bridge](#set-up-ament-workspace).
+
+##### Option 2: Link Conda Environment into ROS (Advanced).
+
+If you are using a conda environment, you can link the conda environment into ROS. This is a bit more advanced, but can be useful if you want to keep your ROS and conda environments separate.
+
+Install using the installation script, but using the `--cpu` flag for a CPU-only installation:
+
+```bash
+./install.sh --cpu
+```
+
+Then, activate the conda environment:
+
+```bash
+conda activate stretch_ai_$VERSION_cpu
+```
+
+Then, [link the package into your ament workspace](#set-up-ament-workspace) and install the package:
+
+```bash
+colcon build --cmake-args -DPYTHON_EXECUTABLE=$(which python)
+```
+
+Some ROS python repositories might be missing - specifically `empy` and `catkin_pkg`. You can install these with:
+
+```bash
+python -m pip install empy catkin_pkg
+```
+
+##### Set Up Ament Workspace on the Robot
```

Comment:
`Setting up Ament Workspace` section heading should be bigger since it is an important step and is being referenced in two other places. 
```suggestion 
#### Set Up Ament Workspace on the Robot
```